custom/src/main/java/co/elastic/otel/config/DynamicInstrumentation.java

```diff
@@ -130,7 +118,7 @@ private static InstrumentationScopeInfo getInstrumentationScopeInfo(Tracer sdkTr
   //                    getTracerConfig(sdkTracer.getInstrumentationScopeInfo())));
   // where SdkTracer.updateTracerConfig(TracerConfig tracerConfig) is equivalent to
   //  this.tracerEnabled = tracerConfig.isEnabled();
-  private static void updateTracerConfigurations(TracerProvider provider) {
+  static void updateTracerConfigurations(TracerProvider provider) {
```

Comment:
All public methods have been refactored to the DynamicConfiguration class, this class only holds methods that should ultimately disappear when the SDK APIs become non-experimental